### PR TITLE
added global context wrapper

### PR DESCRIPTION
**Pull request by suhmantha1**

### CODE REVIEW

File: components/Slices/index.js [hunk 1]
```diff
@@ -23,6 +24,8 @@ const Slices = ({ slices }) => {
         }
         return <div key={i}>{Component}</div>
       })}
+
+      <div>GLOBAL DATA: {useGlobalContext().todoData}</div>
```
Comment (bretmorris) [inline components/Slices/index.js]: This should actually be written like any other React hook

```
const { todoData } = useGlobalContext()
```

This use `todoData` here simply with `{todoData}`

**Comment by bretmorris** [comment]
This looks pretty good but let's test it out on NYHS for a bit before we merge this into the base.

**Comment by suhmantha1** [comment]
@bretmorris updated that comment, and it's working out on nyhs

**Comment by suhmantha1** [comment]
@bretmorris would recommend this global hook

**Pull request closed without merging**